site/app/templates/officeHoursQueue/QueueHistory.twig

```diff
@@ -84,3 +92,37 @@
     </tbody>
   </table>
 </div>
+
+<script>
```

Comment:
It would be better to move this to a dedicated JS file like `site/public/js/office-hours-queue.js`.